src/OpenCensus.Collector.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -67,7 +67,7 @@ public override void OnStartActivity(Activity activity, object payload)
 
             if (span != null)
             {
-                // Note, route is missing at this stage. It will be available later
+                // TODO: Note, route is missing at this stage. It will be available later
```

Comment:
it is actually Note, not a TODO =). route information will be added in separate callback

Reply:
done